maestro-android/src/main/java/dev/mobile/maestro/handlers/LocaleSettingHandler.kt

```diff
@@ -0,0 +1,68 @@
+package dev.mobile.maestro.handlers
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.res.Configuration
+import android.os.Build
+import android.util.Log
+import java.lang.reflect.InvocationTargetException
+import java.util.*
+
+class LocaleSettingHandler(context: Context) : AbstractSettingHandler(context, listOf(CHANGE_CONFIGURATION)) {
+    fun setLocale(locale: Locale) {
+        try {
+            if (hasPermissions()) {
+                setLocaleWith(locale)
+            }
+        } catch (e: Exception) {
+            Log.e(TAG, "Failed to set locale", e)
```

Comment:
Will this be properly propagated in Maestro Cloud?

Reply:
should be part of `adb logcat` @amanjeetsingh150 cmiiw

Reply:
Yes should be in logcat

Reply:
Sorry if this is a stupid question, but does it being in logcat mean that we can correctly surface client errors in the event of misconfigured locale and infra errors in the event of problems on our side?

Reply:
adb logs are served on MaestroCloud, so later clients can use them to debug the issue